projects/rawkode-academy/web/src/pages/blog/[...slug].astro

```diff
@@ -48,7 +48,7 @@ const { Content } = await article.render();
 								href="#"
 								rel="author"
 								class="text-xl font-bold text-gray-900 dark:text-white"
-								>Rawkode Academy
+								>{article.data.authors}
```

Comment:
This will need a .join(",") and a default value if none exists

Reply:
Add in a join. The default case looks like it is handled in `/src/content/config.ts'